src/category_theory/limits/shapes/equalizers.lean

```diff
@@ -156,4 +171,51 @@ def cofork.of_cocone
 @[simp] lemma cofork.of_cocone_ι {F : walking_parallel_pair.{v} ⥤ C} (t : cocone F) (j) :
   (cofork.of_cocone t).ι.app j = eq_to_hom (by tidy) ≫ t.ι.app j := rfl
 
+variables (f g)
+
+section
+variables [has_limit (parallel_pair f g)]
+
+abbreviation equalizer := limit (parallel_pair f g)
+
+abbreviation equalizer.ι : equalizer f g ⟶ X :=
+limit.π (parallel_pair f g) zero
+
+abbreviation equalizer.w : equalizer.ι f g ≫ f = equalizer.ι f g ≫ g :=
+begin
+  erw limit.w (parallel_pair f g) walking_parallel_pair_hom.left,
+  erw limit.w (parallel_pair f g) walking_parallel_pair_hom.right
+end
+
+abbreviation equalizer.lift {W : C} (k : W ⟶ X) (h : k ≫ f = k ≫ g) : W ⟶ equalizer f g :=
+limit.lift (parallel_pair f g) (fork.of_ι k h)
+end
+
+section
+variables [has_colimit (parallel_pair f g)]
+
+abbreviation coequalizer := colimit (parallel_pair f g)
+
+abbreviation coequalizer.π : Y ⟶ coequalizer f g :=
+colimit.ι (parallel_pair f g) one
+
+abbreviation coequalizer.w :  f ≫ coequalizer.π f g = g ≫ coequalizer.π f g :=
```

Comment:
Idem dito: Can/should we `reassoc` this abbreviation?

Reply:
As above.